### DISCUSSION TITLE
Allow iterations to have undefined due dates

### DISCUSSION THREAD

**Discussion by DNin01** [community/community#69961] [Projects and Issues]
### Select Topic Area

Product Feedback

### Feature Area

Projects

### Body

Let's assume you're using iteration fields to track for which version of software each to-do will be released, so you'll have an iteration for every notable release. Sometimes you don't always know exactly when you're going to be ready to release an update or don't want to be particularly strict about when the next update should be released. However, it seems to me like each iteration currently requires a due date, but for less tight release schedules, it can be incredibly helpful to not have to set one right away.

A workaround is setting the iteration's due date to a time far in the future, but it still looks like an actual due date.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by taylorhadden** [comment]
I also think this would be valuable. I would love to be able to specify iterations by version rather than some arbitrary date. My github projects are side projects and are subject to the winds of my free time.
